libs/base/utilities/src/injectables/i18n/i18n.injectable.ts

```diff
@@ -91,3 +91,8 @@ type InferI18nFilterName<F> = F extends I18nContextFilter<infer N, object>
 type InferI18nFilterConfig<F> = F extends I18nContextFilter<string, infer C>
   ? C
   : never;
+
+export interface I18nContent {
```

Comment:
what about possibility to handle all cases with I18nContent, so also raw text (that won't be translated)? something like `raw?: string` ?

Reply:
Nice idea. Tbh, we've already had TextResource interface, I forgot about it, I deprecated it and it should be replaced  with I18nContent

Reply:
forgot about it 🤦:)